docs/how-to/ccv0.sh

```diff
@@ -175,6 +183,9 @@ check_out_repos() {
         git checkout ${tests_branch}
     fi
     git reset --hard origin/${tests_branch}
+
+    source "${BATS_TEST_DIRNAME}/lib.sh"
```

Comment:
I think this line can be removed?

Reply:
The reason we need to source `lib.sh` in this function as well as globally is because the repo is checked out in this function and the code in `lib.sh` is then referenced before exiting the function :)

Reply:
Hey James, thanks so much for the reviews. As Megan says I think we still need both of the `source`s as they run under different circumstances, depending on whether the code has already been checked out or not:
- The first one: `[ -d "${BATS_TEST_DIRNAME}" ] && source "${BATS_TEST_DIRNAME}/lib.sh"` will only run if the code has already been checked out (otherwise it isn't there to source!)
- This one runs in `check_out_repos()`, which will run for a fresh environment when the script hasn't previously run, but not be run most of the time when the tests repo has already been checked out
Together they should provide coverage of the two different states. I hope this makes sense. If you have a better suggestion to solve this then that would be great though!